content/spaces/observability.md

```diff
@@ -1,339 +1,78 @@
 ---
 title: Observability
 weight: 200
-description: A guide to Observability in an Upbound Space
+description: A guide for how to use the integrated observability pipeline feature in a Space.
 ---
 
-## Environment metrics
+{{< hint "important" >}}
+This feature is in preview, requires Spaces `v1.3.0`, and is off by default. To enable, set `features.alpha.observability.enabled=true` when installing Spaces:
 
-<!-- vale off -->
-Upbound recognizes that individual operators have preferences about
-how to monitor their infrastructure. Metrics for Crossplane, Environments,
-and Spaces are exposed using standard Prometheus practices
-<!-- vale on -->
-
-The following components expose Prometheus metrics within the environment
-scope:
-- kube-state-metrics: information about managed resources
-- mxp-gateway: metrics about the ingress into a control plane
-- otel-collector: single endpoint for observing inside the Space
-
-This guide explains how you can use a reference Grafana dashboard built by Upbound to aid in monitoring a Spaces installation.
-
-{{<img src="spaces/images/spaces-dashboard.png" alt="The reference Grafana dashboard built by Upbound for Spaces" >}}
+```bash
+up space init --token-file="${SPACES_TOKEN_PATH}" "v${SPACES_VERSION}" \
+  ...
+  --set "features.alpha.observability.enabled=true" \
```

Comment:
This requires the open telemetry operator to be available on the cluster. 
We will eventually integrate it to `up space init`, but until then it needs to be installed manually:

```bash
# install cert-manager which is a dependency for open telemetry operator
kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.3/cert-manager.yaml
kubectl wait deployment -n cert-manager cert-manager-webhook --for condition=Available=True --timeout=360s

# install open telemetry operator
kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/download/v0.96.0/opentelemetry-operator.yaml
```